components/map/Clusters.module.css

```
@@ -0,0 +1,39 @@
+.small {
+	background-color: rgba(181, 226, 140, 0.6);
+}
+.small div {
+	background-color: rgba(110, 204, 57, 0.6);
+}
+
+.medium {
+	background-color: rgba(241, 211, 87, 0.6);
+}
+.medium div {
+	background-color: rgba(240, 194, 12, 0.6);
+}
+
+.large {
+	background-color: rgba(253, 156, 115, 0.6);
+}
+.large div {
+	background-color: rgba(241, 128, 23, 0.6);
+}
+
+.marker-cluster {
+	background-clip: padding-box;
+	border-radius: 20px;
+	z-index: 100;
+}
+.marker-cluster div {
+	width: 30px;
+	height: 30px;
+	margin-left: 5px;
+	margin-top: 5px;
+
+	text-align: center;
+	border-radius: 15px;
+	font: 12px "Helvetica Neue", Arial, Helvetica, sans-serif;
+}
+.marker-cluster span {
+	line-height: 30px;
+}
```

I'm not sure how this passed linting, but shouldn't there be a new line?

```suggestion
}

```